### PR TITLE
Persistent menu

### PR DESCRIPTION
**Pull request by ahendy0**

### CODE REVIEW

File: example/example.py [hunk 1]
```diff
@@ -86,9 +86,30 @@ def webhook():
             msg = Message('text', payload)
             bot.send_message(msg, recipient)
 
+
+
```
Comment (ben-cunningham) [inline example/example.py]: le extra newlines

File: fbmsgbot/bot.py [hunk 2]
```diff
@@ -73,3 +73,33 @@ def message_from_reciept(receipt):
...
 
         raise NotImplementedError
+
+    def set_persistent_menu(self, buttons):
+        """
+        Sets the persistent menu. 
+        @params: buttons -> list of button objects
+        @moreinfo: https://developers.facebook.com/docs/messenger-platform/thread-settings/persistent-menu
+        """
+        MAX_BUTTONS = 5
```
Comment (ben-cunningham) [inline fbmsgbot/bot.py]: Is this set by Facebook?

Comment (ahendy0) [inline fbmsgbot/bot.py]: @ben-cunningham  yeap

**Comment by ahendy0** [comment]
@ben-cunningham also i like the syntax for bot.py set_persistent_menu function descriptor

**Comment by ben-cunningham** [comment]
Yeah i concur it's good

File: example/example.py [hunk 3]
```diff
@@ -86,9 +86,30 @@ def webhook():
...
+        payload="My defined payload string",
+    )
+    
+    bot.set_welcome("Welcome to PyBot!")
+    bot.set_persistent_menu([
+        web_button,
+        postback_button,
+    ])
+    
+
```
Comment (ben-cunningham) [inline example/example.py]: extra new line

**Comment by ahendy0** [comment]
Rip rebase

**Pull request merged**